### PR TITLE
Add 1.0em bottom margin to sidebar content boxes and a surrounding...

### PR DESCRIPTION
**Pull request by ARBaynes**
... div to the current sidebar-nav

@punalsc thank you for your advice on this :)

### CODE REVIEW

**Comment by punalsc** [comment]
Hi @Urborging,

Here are some of the scss i would suggest changing.
```scss
#sidebar {
  > div {
    margin-bottom: 1.5em;

    &:last-of-type {
      margin-bottom: 0;
    }
  }
}
```

**Comment by punalsc** [comment]
Hi @Urborging ,
@domingobishop made some suggestions which would makes sense, and i have suggested these changes, in the comments

File: sidebar.php [hunk 1]
```diff
@@ -1,51 +1,58 @@
...
-				<?php }
-			} ?>
-		</ul>
-	</div>
+    <!-- resource ratings start -->
+	<?php if (function_exists("display_resource_ratings_tna_base")) {
+		display_resource_ratings_tna_base();
+	} ?>
+    <!-- resource ratings end -->
+    <div>
```
Comment (punalsc) [inline sidebar.php]: add a class to the div `<div class="sidebar-item-wrapper">`

File: css/sass/_sidebar.scss [hunk 2]
```diff
@@ -1,7 +1,6 @@
 /* TNA sidebar styles */
 
 aside {
-  margin-bottom: 1.5em;
   .sidebar-header {
```
Comment (punalsc) [inline css/sass/_sidebar.scss]: using the new class created add the sass directly after the aside.
```scss
> .sidebar-item-wrapper {
    margin-bottom: 1.5em;
  }
```

File: sidebar.php [hunk 3]
```diff
@@ -1,51 +1,58 @@
...
-					<li>
-						<a href="<?php echo make_path_relative( get_page_link( $page->ID ) ); ?>" title="<?php echo $page->post_title ?>">
-							<?php echo $page->post_title; ?>
-						</a>
-					</li>
-				<?php }
-			} ?>
-		</ul>
-	</div>
+    <!-- resource ratings start -->
```
Comment (punalsc) [inline sidebar.php]: Wrap your function inside of wrapper if you want to test it `<div class="sidebar-item-wrapper">`

File: css/sass/_sidebar.scss [hunk 4]
```diff
@@ -50,4 +51,14 @@ aside {
   .sidebar-item h3 {
     margin-top: 0;
   }
-}
\ No newline at end of file
+}
+
+#sidebar {
```
Comment (punalsc) [inline css/sass/_sidebar.scss]: Delete this section, as it's not needed.

**Comment by domingobishop** [comment]
@punalsc I'm a little confused with this feature. Are we planning to merge this into Develop? If so, we need to do an extensive impact review and testing on how this will impact on child sites. In particular ones that have custom sidebars.

**Comment by punalsc** [comment]
@domingobishop,
As discussed, we need to make some changes to tna-base to accommodate pages with more then one sidebar items. If you can advice how to add a custom sidebar, we can test it locally, then make a request to deploy to test, if that works?

**Comment by domingobishop** [comment]
@punalsc Ok, I was under the assumption this was an experiment for course work? For deployment, any new sidebars should be created as custom sidebars in the child theme. The master sidebar shouldn't be altered to cater for one specific need. Let's talk about this tomorrow.

**Comment by domingobishop** [comment]
@punalsc for example, the child blog theme has a sidebar which is exactly what you are trying to achieve: https://github.com/nationalarchives/tna-child-blog
My feeling is this kind of customisation should be done in child themes. But lets look at this again tomorrow.

**Comment by domingobishop** [comment]
@ARBaynes Should we close this PR?

**Comment by ARBaynes** [comment]
Sure, it's past my work-based project submission so this is no longer of use

**Pull request closed without merging**